.github/workflows/easy-cp.yml

```diff
@@ -0,0 +1,81 @@
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+name: Cherry-pick Labeled PR to Release Branch
+
+on:
+  pull_request:
+    branches: master
+    types: [labeled]
+
+permissions: write-all
+
+jobs:
+  cherrypick_to_release:
+    name: cherrypick_to_release
+    runs-on: ubuntu-latest
+    if: |
+      (github.event.label.name == format('cp{0} beta', ':') || github.event.label.name == format('cp{0} stable', ':')) &&
+      (github.event.pull_request.merged == true)
+    steps:
+      - name: Get Release Channel
+        run: |
+          echo "CHANNEL=$(echo ${{ github.event.label.name }} | cut -d ':' -f 2 | xargs)" >> $GITHUB_ENV
```

Comment:
Is the double echo unnecessary?

export CHANNEL=$(${{ github.event.label.name }} | cut -d ':' -f 2)

Reply:
Tested and my understanding is that both the inner and outer echos are needed.

My understanding is that the inner `echo` is needed to pipe it to the next expression.
without the inner echo, `$word | cut -d ':' -f 2` would throw command not found error on $word
with the echo, `echo $word | cut -d ':' -f 2` would pipe the output of `echo $word` into the next expression

The outer `echo` is needed due to the unique way of storing environment variables in github actions.
without the outer echo and using `export $var`, the variable is not stored in github actions. [failed workflow](https://github.com/XilaiZhang/miscellaneous-side-project/actions/runs/7106918754/job/19347336352)
with the outer echo, Echo it into GITHUB_ENV would prevserve it. (https://stackoverflow.com/questions/57968497/how-do-i-set-an-env-var-with-a-bash-expression-in-github-actions)

Reply:
Umm yeah I haven't found a source that includes all the supported syntax. Currently my learnings come from failed workflows and stackoverflow.

Reply:
I found https://docs.github.com/en/actions/learn-github-actions/essential-features-of-github-actions which covers the basics